dts/arm/infineon/psoc6/common/gpio_pins_def.h

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
please split https://github.com/zephyrproject-rtos/zephyr/commit/87a40b25a2f773027a7013fb435212defe4fe140 into smaller chunks, to ease review process. I can spot a few issues, but I can't hardly review thousands of line in one go.